developers/weaviate/model-providers/anthropic/generative.md

```diff
@@ -236,6 +258,29 @@ In other words, when you have `n` search results, the generative model generates
 
 </Tabs>
 
+### RAG with images
+
+You can also use images when querying Weaviate and perform retrieval augmented generation in both single prompts and grouped tasks. 
```

Comment:
Wdyt about:

`You can also supply images as a part of the input when performing ...`